src/app-layout/styles.scss

```diff
@@ -7,6 +7,30 @@
 @use '../internal/styles/tokens' as awsui;
 @use './constants' as constants;
 
+/* stylelint-disable selector-combinator-disallowed-list, selector-max-type, selector-max-universal */
+.resize {
+  &-active * {
+    user-select: none;
+  }
+  &-active iframe {
+    // Prevents iframes from eating the mouseup event if the cursor is on top of it.
+    pointer-events: none;
+  }
+  &-side * {
+    cursor: ew-resize;
+  }
+  &-bottom * {
+    cursor: ns-resize;
+  }
+}
```

Comment:
There should be `/* stylelint-enable selector-combinator-disallowed-list, selector-max-type, selector-max-universal */` at the end of the section to finish the exceptional part